nilearn/image/image.py

```diff
@@ -610,6 +610,21 @@ def index_img(imgs, index):
      >>> single_mni_image = index_img(joint_mni_image, 1)
      >>> print(single_mni_image.shape)
      (91, 109, 91)
+
+    We can also select multiple frames using the `slice` constructor::
+
+     >>> functional_scan = datasets.fetch_development_fmri(
+     ...    n_subjects=1, verbose=0).func # doctest: +ELLIPSIS
```

Comment:
it is better not to download whole datasets in the doctests, especially the less reliable ones. these example snippets are just meant to illustrate the function signature, you can use fake data or data that is distributed with nilearn -- e.g a list of mni template copies as above

Reply:
I found that a list of MNI templates was less clear on read-through. I'd love to illustrate this with a functional dataset, though it looks like we don't ship any. Do you have recommendations of a dataset you'd prefer over development_fmri ?

Reply:
the previous example in the same docstring uses the template so I would stick with it. otherwise any image that does not need to be downloaded is ok. I agree that it would be nice to ship one example functional image for such use cases: https://github.com/nilearn/nilearn/issues/2184